pdns/dnsdist-idstate.hh

```diff
@@ -271,4 +272,5 @@ struct IDState
   bool destHarvested{false}; // if true, origDest holds the original dest addr, otherwise the listening addr
   bool dnssecOK{false};
   bool useZeroScope{false};
+  std::unique_ptr<PacketBuffer> xskPacketHeader;
```

Comment:
Just a nit but this should go with the other unique_ptr to prevent losing space due to padding.

Reply:
Done.